src/Resources/ElasticSearch/ElasticIndexerResource.php

```diff
@@ -24,7 +24,7 @@ class ElasticIndexerResource extends Resource
      */
     public function getURI()
     {
-        return 'elasticsearch';
+        return 'elasticindexer';
```

Comment:
It should be described, what version should be used with that changes

Reply:
ok. after creating the version in ms-indexer I am adding in this description number of this version.